sqlchain/chain.go

```diff
@@ -1272,5 +1074,32 @@ func (c *Chain) billing(node *blockNode) (ub *types.UpdateBilling, err error) {
 		i++
 	}
 	ub.Receiver, err = c.databaseID.AccountAddress()
+	ub.Range.From = uint32(minHeight)
+	ub.Range.To = uint32(h)
 	return
 }
+
+func (c *Chain) SetLastBillingHeight(h int32) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method Chain.SetLastBillingHeight should have comment or be unexported